plugins/webhooks/index.js

```diff
@@ -83,7 +83,7 @@ function startPRJob(options, request) {
             });
         })
         // create a build
-        .then(event => buildFactory.create({ jobId, sha, username, eventId: event.id }))
+        .then(event => buildFactory.create({ jobId, sha, username, eventId: event.id, prRef }))
```

Comment:
I don't think this would work. `prRef` is not an attribute of `build`. It won't get saved.